src/layout/DetailRoute.js

```diff
@@ -41,23 +43,24 @@ const Layout = ({ children }) => {
           {/* NFT Image */}
           <div className={styles.nft__image__outer__wrapper}>
             <div className={styles.nft__image__wrapper} style={{width:"60%"}}>
-              {fileType.toLowerCase() === "mp4" ?
+              {fileType?.toLowerCase() === "mp4" ?
                   <video
                       style={{width: '100%', borderRadius: "8px"}}
-                      src={nft__detail?.image}
+                      src={nft__detail?.file_url}
+                      controls
                   />
-                  : fileType.toLowerCase() === "mp3" ?
+                  : fileType?.toLowerCase() === "mp3" ?
                       (
                           <div style={{width:"100%",padding:"0 2px"}}>
                             <audio style={{width:"inherit",marginTop:"60px"}} controls>
-                              <source src={nft__detail?.image}/>
+                              <source src={nft__detail?.file_url}/>
                             </audio>
                           </div>
                       ) :
                       (
                           <img
-                              src={nft__detail?.image}
-                              alt={nft__detail.name}
+                              src={nft__detail?.file_url}
+                              alt={nft__detail?.title}
                           />
                       )}
```

Comment:
Please move these conditional code to a separate component as it being used in other component as well